test/docfx.Test/common/TestFramework.cs

```diff
@@ -25,6 +26,7 @@ public TestFramework(IMessageSink messageSink)
 
         protected override ITestFrameworkExecutor CreateExecutor(AssemblyName assemblyName)
         {
+            Environment.SetEnvironmentVariable("DOCFX_APPDATA_PATH", Path.GetFullPath("appdata"));
```

Comment:
>SetEnvironmentVariable [](start = 24, length = 22)

What about fetch the original value first, and set it back when all tests finish?

Reply:
This method sets the process wide environment variable, it does not affect your machine and other processes.

---
In reply to: [203579305](https://github.com/dotnet/docfx/pull/3094#discussion_r203579305) [](ancestors = 203579305)